src/ExExpression.elm

```diff
@@ -139,10 +139,18 @@ elixirTypeInstances c e =
                     name ++ ".0"
 
         Character value ->
-            if value == ' ' then
-                "?\\s"
-            else
-                "?" ++ String.fromChar value
+            case value of
+                ' ' ->
+                    "?\\s"
+
+                '\n' ->
+                    "?\\n"
+
+                '\t' ->
+                    "?\\t"
+
+                other ->
```

Comment:
Also need to escape `\r`.  Unsure if any others are absolutely required, but `\r` is for sure.